force-app/main/default/classes/Homework.cls

```diff
@@ -77,14 +193,44 @@ public with sharing class Homework
     //          the returned list would be 8,9,10,1,2,3,4,5,6,7
     public List<Integer> rotateFinalThree(List<Integer> values)
     {
+        List<Integer> rotatedFinalThree = new List<Integer>();
+        
+        // have assumed values must cotain 10 Integers
+        List<Integer> firstSixValues = new List<Integer>();
+        List<Integer> lastThreeValues = new List<Integer>();
+
+        for (Integer i = 0; i < values.size(); i++)
+        {
+            if (i < 7)
+            {
+                firstSixValues.add(values[i]);
+            }
+            else
+            {
+                lastThreeValues.add(values[i]);
+            }
```

Comment:
This approach works, but you could be a little more efficient and offset the index (i) by the starting point (7), e.g.

```
        Integer startPos=7;
        for (Integer idx=0; idx<values.size(); idx++)
        {
            rotated.add(values[Math.mod(idx+startPos, 10)]);
        }

```
This will start at position 7 and use the mod function to restart at the beginning when it goes over 10.